libs/auth-api-lib/src/lib/services/delegations.service.ts

```diff
@@ -352,23 +363,126 @@ export class DelegationsService {
       (!hasDelegationTypeFilter ||
         delegationTypes?.includes(DelegationType.Custom))
     ) {
-      delegationPromises.push(this.findAllValidCustomIncoming(user))
+      needVerifyDelegationPromises.push(this.findAllValidCustomIncoming(user))
     }
     if (
       (!client || client.supportsPersonalRepresentatives) &&
       (!hasDelegationTypeFilter ||
         delegationTypes?.includes(DelegationType.PersonalRepresentative))
     ) {
-      delegationPromises.push(this.findAllRepresentedPersonsIncoming(user))
+      needVerifyDelegationPromises.push(
+        this.findAllRepresentedPersonsIncoming(user),
+      )
```

Comment:
Afaik, representedPersons delegations are not stored in the delegation table, so we need different logic to "disable" them. 

I see that the invalidateDelegations function skips over those delegations (without `id`), and they get removed from the response, which is maybe a fine MVP. But there is still the matter of the audit message which would sound repeatedly if there is a deceased represented person.

Reply:
I divided the logic up to make things more clear.
now these two methods, i.e. `findAllRepresentedPersonsIncoming` and `findAllValidCustomIncoming` encapsulate the logic. That is they handle deleting the appropriate models